packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -21,27 +23,59 @@ import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
 
 export enum DestinationAddressErrors {
   INVALID_ADDRESS = 'The destination address is not a valid address.',
-  REQUIRED_ADDRESS = 'The destination address is required.'
+  REQUIRED_ADDRESS = 'The destination address is required.',
+  DENYLISTED_ADDRESS = 'The address you entered is a known contract address, and sending funds to it is likely to result in losing said funds. If you think this is a mistake, contact support@arbitrum.io.'
```

Comment:
```suggestion
  DENYLISTED_ADDRESS = 'The address you entered is a known contract address, and sending funds to it would likely result in losing said funds. If you think this is a mistake, file a <ExternalLink
              href="https://support.arbitrum.io/hc/en-us/requests/new"
              className="arb-hover underline"
            >
              support ticket
            </ExternalLink>.
```
^ don't directly commit this but we need sth like that as the email has been deprecated and emails sent to this address will be undelivered